### PR TITLE
Documentation refresh

### PR DESCRIPTION
**Pull request by vodorok**

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4827097&bid=16041147)

File: readme.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Run `mvn -f mavendeps/pom.xml p2:site && mvn clean verify` in the root of the pr
 ### Install
 
 * Add the generated update site thats located under `path/to/cloned/project/releng/org.codechecker.eclipse.update/target/repository/` to `Help -> Install New Software...` in Eclipse
+  * Alternatively, extract the release downloaded from Releases page, and use that as an update site.
```
Comment (gyorb) [inline readme.md]: Wouldn't github just compress the repository ad a zip file? Can that be used as an update site?

File: readme.md [hunk 2]
```diff
@@ -50,23 +51,23 @@ The plugin is activated on a per project basis, first you have to add the __Code
 ![CodeChecker Nature Add](docs/nature.png)
 
 #### 3. Configure CodeChecker and checkers
-After that, the settings can be customized in the project preferences window under the CodeChecker panel. For the plugin to work correctly, it is mandatory to add the correct path to the root of the CodeChecker package. You can download and compile CodeChecker from [here](https://github.com/Ericsson/codechecker).
+
+After the __CodeChecker Nature__  is added to the the project, the plugin can be configured globally in `Window -> Preferences -> CodeChecker` panel. Or for the individual project, from the `right click context menu -> Properties -> CodeChecker` page.
```
Comment (gyorb) [inline readme.md]:
```suggestion
After the __CodeChecker Nature__  is added to the the project, the plugin can be configured globally in `Window -> Preferences -> CodeChecker` panel, or for the individual project, from the `right click context menu -> Properties -> CodeChecker` page.
```

File: readme.md [hunk 3]
```diff
@@ -50,23 +51,23 @@ The plugin is activated on a per project basis, first you have to add the __Code
...
+In the first section, you select how and which CodeChecker will be used. After that set some analysis related settings. You should build CodeChecker in a __standalone package__ configuration, and add it to the PATH environment variable, and use the Search in PATH option. Alternatively you can specify a different instance with the Pre built package option. But be aware that the plugin not supports virtual environment, that CodeChecker needs in the default configuration. You can download and compile CodeChecker from [here](https://github.com/Ericsson/codechecker).
```
Comment (gyorb) [inline readme.md]: I think an example would be nice here. I did not found any documentation in the codechecker repo about the standalon package mode (maybe I missed it).
```sh
cd codechecker
make standalone_package
export PATH=$PWD/build/CodeChecker/bin:$PATH
```

**Review by gyorb — CHANGES_REQUESTED** [review]

**Comment by Vodorok** [comment]
Thanks @gyorb I've clarified the docs and incorporated your notices.

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4827097&bid=16041147)

**Pull request merged**